exercises/practice/simple-linked-list/src/lib.rs

```diff
@@ -58,8 +58,8 @@ impl<T> FromIterator<T> for SimpleLinkedList<T> {
 // of IntoIterator is that implementing that interface is fairly complicated, and
 // demands more of the student than we expect at this point in the track.
 
-impl<T> Into<Vec<T>> for SimpleLinkedList<T> {
-    fn into(self) -> Vec<T> {
+impl<T> From<SimpleLinkedList<T>> for Vec<T> {
+    fn from(mut _linked_list: SimpleLinkedList<T>) -> Vec<T> {
```

Comment:
Kind of surprised rustc/CI didn't issue a warning about an unnecessary `mut` for this line.

Reply:
Ah, so it did when I kept it as `linked_list`. So I'm guessing no lints when you have the variable ignored from linting?

Reply:
That's really interesting, and IMO potentially a bug. Not a huge priority for anyone though, I imagine.